x-pack/plugins/triggers_actions_ui/public/plugin.ts

```diff
@@ -80,7 +88,13 @@ export interface TriggersAndActionsUIPublicPluginStart {
   getEditAlertFlyout: (
     props: Omit<RuleEditProps, 'actionTypeRegistry' | 'ruleTypeRegistry'>
   ) => ReactElement<RuleEditProps>;
+  // getAlertHooks: () => {
```

Comment:
Do we need to export these hooks?

Reply:
at one point, we will like to do it. Does not have to be in this PR. Let's keep this PR simple.